packages/ndla-ui/src/model/WordClass.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const WordClass = {
```

Comment:
Egentlig må jo alle verdiene her skrives med bindestrek for å matche verdier i https://github.com/NDLANO/backend/blob/master/concept-api/src/main/scala/no/ndla/conceptapi/model/domain/Gloss.scala